src/com/sap/piper/ConfigurationHelper.groovy

```diff
@@ -167,6 +167,15 @@ class ConfigurationHelper implements Serializable {
         return this
     }
 
+    ConfigurationHelper withPropertyInList(String key, Set values){
```

Comment:
... it is not a list anymore in the signature, but a set, but it is still named `withPropertiesInList` as far was I can see. Sounds weird having "list" the method name when we do not deal with a list. Maybe something like `withPropertyWithValues(./.)`, `withPropertyHavingValue(./.)`, `withPropertyValueIn(./.)`. That names are not the yellow part of the egg so to say :-), maybe we find an even better name.

Reply:
Good point. I like withPropertyWithValues most. I will use that one.